web/crux-ui/src/models/registry.ts

```diff
@@ -308,6 +309,7 @@ export const editableRegistryToDto = (ui: EditableRegistry): CreateRegistryDto =
       const details: UpsertDetailsDto<V2RegistryDetailsBase> = {
         url: ui.url,
         public: ui.public,
+        imageNamePrefix: !ui.imageNamePrefix || ui.imageNamePrefix.trim() === '' ? null : ui.imageNamePrefix,
```

Comment:
This "null or empty" comparison seems generic, might make a fine helper function (optional, just a note).